.github/workflows/ci-tests.yml

```diff
@@ -25,16 +49,14 @@ jobs:
         xcode-version: ${{ env.XCODE_VERSION }}
     - name: Checkout Repo
       uses: actions/checkout@v3
-    - name: Setup Repo
-      id: setup-repo
-      uses: ./.github/actions/setup-repo
     - name: Run Swift Build
       shell: bash
       run: |
         cd ${{ matrix.package }} && swift build
 
   build-and-unit-test:
     runs-on: macos-13
+    needs: tuist-generation
```

Comment:
`needs` is a confusing property name for this; should be `required-first` or `after`.

Reply:
yea not sure how they landed on that name